docs/transparent-validation/lab-assembly-guides/msi_z690/validation.md

```diff
@@ -0,0 +1,109 @@
+# MSI PRO Z690-A WIFI DDR4 Lab Setup
+
+## Introduction
+
+This document describes assembly of MSI PRO Z690-A DDR4 lab setup. At this
+point fully assembled platform and RTE should be ready.
+
+## Requirements
+
+| Part              | Model Name                                                    | Image                                    |
+|:------------------|:-------------------------------------------------------------:|:----------------------------------------:|
+| Device Under Test | MSI PRO Z-690A DDR4 - [Assembled](assembly.md)                |![Motherboard](../images/motherboard.jpg)    |
+| RTE               | RTE - [Assembled](../../rte/introduction.md#rte-introduction) |![RTE](../images/rte_built.jpg)              |
+| Sonoff            | Sonoff - [Assembled](../../sonoff/sonoff-setup.md)            |![Sonoff](../images/sonoff_disconnected.jpg) |
+| cables            | standard female-female connection wire 2.56mm raster          |![cables](../images/female_female_cables.jpg)|
+
```

Comment:
Okay, there's a list of needed hardware, but is there a need to prepare it in some way before starting the mounting procedure? We should add here links to the documentation that describes the preparation procedure for all of the devices.

Reply:
Links are already included in the table